test-clients/src/main/resource/contract/solidity/hip-206/IHederaTokenService.sol

```diff
@@ -428,6 +428,31 @@ interface IHederaTokenService {
         external
         returns (int64 responseCode, bool defaultKycStatus);
 
+    /// Query if token account has kyc granted
+    /// @param token The token address to check
+    /// @param account The account address associated with the token
+    /// @return responseCode The response code for the status of the request. SUCCESS is 22.
+    /// @return kycGranted True if `account` has kyc granted for `token`
+    function isKyc(address token, address account)
```

Comment:
LGTM ! Other than the name of this function should be `isKycGranted` imo

Reply:
This name is used in [smart-contratcts-repo](https://github.com/hashgraph/hedera-smart-contracts/blob/main/hts-precompile/IHederaTokenService.sol#L581) and HIP 514. Do we want to change it?

Reply:
@mustafauzunn I see, I would defer to @shemnon. If that name was as per discussion , I am fine with it.